### PR TITLE
[cordova] Update shared mode package method in webview plugin

### PR DESCRIPTION
**Pull request by zhuyongyong**
- Follow ﻿https://github.com/crosswalk-project/cordova-plugin-crosswalk-webview/blob/master/README.md
- Add one way for CrosswalkVersion to install cordova-plugin-crosswalk-webview from npm

Impacted tests(approved): new 0, update 3, delete 0
Unit test platform: Crosswalk Project for Android 15.44.384.7
Unit test result summary: pass 3, fail 0, block 0

### CODE REVIEW

File: usecase/usecase-cordova-android-tests/samples/CrosswalkVersion4.x/res/Untitled Document [hunk 1]
```diff
@@ -0,0 +1,12 @@
+    if BUILD_PARAMETERS.pkgmode == "shared":
```
Comment (Honry) [inline usecase/usecase-cordova-android-tests/samples/CrosswalkVersion4.x/res/Untitled Document]: Please rename this file.

File: usecase/usecase-cordova-android-tests/res/comm.py [hunk 2]
```diff
@@ -12,18 +12,46 @@
...
 from optparse import OptionParser
+script_path = os.path.realpath(__file__)
+const_path = os.path.dirname(script_path)
+tool_path = const_path + "/../tools/"
+plugin_tool = const_path + "/../tools/cordova-plugin-crosswalk-webview/"
 
 def setUp():
     global CROSSWALK_VERSION
     global CROSSWALK_BRANCH
+    global PACK_TYPE
```
Comment (Honry) [inline usecase/usecase-cordova-android-tests/res/comm.py]: Miss CORDOVA_VERSION definition.

File: usecase/usecase-cordova-android-tests/samples/CrosswalkVersion4.x/res/Untitled Document (hunk 1 above)
Comment (Honry) [inline usecase/usecase-cordova-android-tests/samples/CrosswalkVersion4.x/res/Untitled Document]: Useless file?

File: usecase/usecase-cordova-android-tests/samples/CrosswalkVersion4.x/res/test (copy).py [hunk 3]
```diff
@@ -44,14 +44,9 @@
     print "Wrong pkg-arch, only support: %s, exit ..." % PKG_ARCHS
```
Comment (Honry) [inline usecase/usecase-cordova-android-tests/samples/CrosswalkVersion4.x/res/test (copy).py]: test (copy).py? Typo error.

File: usecase/usecase-cordova-android-tests/samples/CrosswalkVersion4.x/res/test.py [hunk 4]
```diff
@@ -0,0 +1,414 @@
...
+        os.system('sed -i "s/<preference name=\\"xwalkVersion\\" value=\\".*/<preference name=\\"xwalkVersion\\" value=\\"org.xwalk:xwalk_shared_library:%s\\" \/>/g" config.xml' % comm.CROSSWALK_VERSION)
+    else:
+        os.system('sed -i "s/<preference name=\\"xwalkVersion\\" value=\\".*/<preference name=\\"xwalkVersion\\" value=\\"org.xwalk:xwalk_core_library:%s\\" \/>/g" config.xml' % comm.CROSSWALK_VERSION)
+    os.system("cordova build android")
+    if BUILD_PARAMETERS.pkgarch == "x86":
+        os.system('cp platforms/android/build/outputs/apk/android-x86-debug.apk ../CrosswalkVersion_stable_14.apk')
+    else:
+        os.system('cp platforms/android/build/outputs/apk/android-armv7-debug.apk ../CrosswalkVersion_stable_14.apk')
+
+    lsstatus = commands.getstatusoutput("ls ../CrosswalkVersion_stable_1.apk")
```
Comment (Honry) [inline usecase/usecase-cordova-android-tests/samples/CrosswalkVersion4.x/res/test.py]: Why not use "for" loop?

File: usecase/usecase-cordova-android-tests/res/comm.py (hunk 2 above)
Comment (zhuyongyong) [inline usecase/usecase-cordova-android-tests/res/comm.py]: This is a CORDOVA_VERSION definition

**Comment by Honry** [comment]
Merge as-is. For new way to install webview plugin from npm, since new npm plugin has not released yet, let's do verification by then.

**Pull request merged**